app/controllers/application_controller.rb

```diff
@@ -11,11 +11,12 @@ class ApplicationController < ActionController::Base
   def authenticate_admin!
     render head: :forbidden, text: 'Forbidden' unless current_user && current_user.admin?
   end
-  
+
   private
 
   def set_gon_globals
     gon.authenticity_token = form_authenticity_token
     gon.locale = I18n.locale
+    gon.vacation_request_statuses = VacationRequest.statuses
```

Comment:
I wasn't sure the plural of *status* was actually *statuses* but it seems it is lol